main.bicep

```diff
@@ -7,6 +7,9 @@ param existingIotHubResourceGroupName string = ''
 @description('(Optional) Resource name of an Azure IoT Hub to reuse.')
 param existingIotHubName string = ''
 
+@description('(Optional) Custom domain name for anomaly detector.')
+param customAnomalyDetectorDomainName string = ''
```

Comment:
This will probably fail the check `Assert-TemplateUIParameters` in `Assert-ARMTemplateCorrectness.ps1`.

Perhaps an argument could be made that customers need not the ability to modify this?